notebooks/ntsb-duckdb.ipynb

```diff
@@ -0,0 +1,340 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyarrow.fs\n",
+    "from ray.data import ActorPoolStrategy\n",
+    "import sycamore\n",
+    "import json\n",
+    "from sycamore.functions.tokenizer import HuggingFaceTokenizer\n",
+    "from sycamore.llms import OpenAIModels, OpenAI\n",
+    "from sycamore.transforms import COALESCE_WHITESPACE\n",
+    "from sycamore.transforms.merge_elements import MarkedMerger\n",
+    "from sycamore.transforms.partition import SycamorePartitioner\n",
+    "from sycamore.transforms.extract_schema import OpenAISchemaExtractor, OpenAIPropertyExtractor\n",
+    "from sycamore.transforms.embed import SentenceTransformerEmbedder"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sycamore.data.document import Document\n",
```

Comment:
Add a comment here explaining what this this cell is. It's used for data cleaning and formatting later in the Sycamore script.